python/xir/program.py

```diff
@@ -479,20 +483,20 @@ def serialize(self, minimize: bool = False) -> str:
             res.extend([f"    {k}: {v};" for k, v in self.options.items()])
             res.append("end;\n")
 
-        res.extend([f"{dec};" for v in self._declarations.values() for dec in v ])
+        res.extend([f"{dec};" for v in self._declarations.values() for dec in v])
```

Comment:
Minor point, but should we be consistent in how to write the short form of "declaration"? I've been using `decl` so far (borrowing from C++ `decltype` and `declval`) but I'm not sure what you prefer:
```suggestion
        res.extend([f"{decl};" for v in self._declarations.values() for decl in v])
```

Reply:
I prefer `decl` so let's stick with that.